📝 ✨ feat: add callback method customization for async v2 API (#43)
gladiaio/docs: snippets/get-transcription-result.mdx

  Callback are HTTP calls that you can use to get notified when your transcripts are ready.

>   Instead of polling and keeping your server busy and maintaining work, you can use the `callback` feature.
>   By providing a `callback_config` field along with a `callback` field set to `true` on your transcription request body payload, we will send the results right there once 
>   your transcription is done.

Reviewer: `By providing...` is a bit redundant with the sentence coming right after the example.  Maybe we can remove it ?
```suggestion  
  Instead of polling and keeping your server busy and maintaining work, you can use the `callback` feature to receive the result to a specified endpoint:
```

  ```json
  {
    "audio_url": "YOUR_AUDIO_URL",
    "callback": true,
    "callback_config": {
      "url": "https://yourserverurl.com/your/callback/endpoint/",
      "method": "POST"
    }
  }
  ```

>   Once the transcription is done, a request will be made to the `callback_config.url` you provided using the `callback_config.method` you provided.

Reviewer: ```suggestion
  Once the transcription is done, a request will be made to the url you provided in `callback_config.url` using the HTTP method you provided in `callback_config.method`.
```
